lambdas/service/app.py

```diff
@@ -1473,6 +1473,9 @@ def start_manifest_generation_fetch():
     return response
 
 
+manifest_service_name = 'manifest'
```

Comment:
Isn't this a lambda name?

Reply:
And why is this needed? Can't we just follow the same pattern (`app.lambda_name(generate_manifest`))?

Reply:
It's not needed. We can follow the same pattern. We discussed this a couple weeks ago which absolved some concerns about performance I had.